### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by nnonickreal** [Pilot1782/bad_copeheimer#85] [Q&A] [answered]
i'm still new to python 
i can't start the bot
Error:
TOKEN = data["TOKEN"]
KeyError: 'TOKEN'

**Comment by Pilot1782** [comment, marked as answer]
Can you paste the output of the terminal like this:

```
C:\Users\****\Downloads\: python3 dis-bot.pyw
...
```

**Reply by nnonickreal** [reply]
![Снимок экрана (26)](https://user-images.githubusercontent.com/116197076/197017452-3d6859fb-fec0-40ef-ac39-7c6419dd53c1.png)

**Reply by Pilot1782** [reply]
Try checking the settings file as explained [here](https://github.com/Pilot1782/bad_copenheimer/wiki/Installation)

**Reply by nnonickreal** [reply]
Checked. Everything is correct

**Reply by nnonickreal** [reply]
I found a fix but now I'm getting a different error:
Sorry, Execution of this file failed.

**Reply by Pilot1782** [reply]
Turn on debugging in the settings, this should give a lot more info. Also, what fix did you end up implementing?

**Reply by nnonickreal** [reply]
im fixed this bug, thx
